### ISSUE TITLE
complete set of DOM facilities

### ISSUE THREAD

**Issue by dherman** [mozilla/task.js#13]
The `task.dom.js` file is just a stub. Fill this out with a more complete set of promise-ified DOM facilities.

(Depends on issue #12.)

**Comment by dherman** [comment]
Out of scope for task.js; these can be provided as promise wrappers with some other framework or library, which would automatically be compatible with task.js.
